packages/nextjs/utils/getFiltersFromQuery.ts

```diff
@@ -14,7 +14,7 @@ export const getFiltersFromQuery = (
         return acc;
       }
 
-      const queryValueIsString = typeof queryValue === "string" || queryValue instanceof String;
```

Comment:
This previous test did nothing as a `queryValue` since the left hand side narrowed out string, so the right hand side could not be an instance of a string.